packages/repository/src/model.ts

```diff
@@ -6,6 +6,8 @@
 import {Options, AnyObject, DataObject} from './common-types';
 import {Type} from './types';
 
+// tslint:disable:no-any
```

Comment:
I find this suspicious, why is it necessary to disable `no-any` rule when there are no other changes made in this file?